### PR TITLE
Certificate based authentication for Android

### PR DESCRIPTION
**Pull request by bhariharan**

### CODE REVIEW

File: libs/SalesforceSDK/res/xml/servers.xml [hunk 1]
```diff
@@ -1,5 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
 <servers>
-  <server name="Production" url="https://login.salesforce.com"/>
-  <server name="Sandbox" url="https://test.salesforce.com"/>
+    <server name="AirWatch" url="https://airwtrial-dev-ed.my.salesforce.com"/>
```
Comment (bhariharan) [inline libs/SalesforceSDK/res/xml/servers.xml]: I'll revert this.

File: libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/OAuthWebviewHelper.java [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2011-2012, salesforce.com, inc.
+ * Copyright (c) 2011-2014, salesforce.com, inc.
```
Comment (wmathurin) [inline libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/OAuthWebviewHelper.java]: 2015 maybe ;-)

Comment (bhariharan) [inline libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/OAuthWebviewHelper.java]: :+1:

**Comment by wmathurin** [comment]
Looks fine otherwise. No issue running it with app configurator / configured app?

**Comment by khawkins** [comment]
Looks fine.

**Comment by bhariharan** [comment]
Haven't run ```AppConfigurator``` yet, since it needs ```FDE```, but I will before merging this in.

**Pull request merged**